src/main/java/org/folio/edge/oaipmh/OaiPmhHandler.java

```diff
@@ -195,7 +217,17 @@ private void handleError(RoutingContext ctx, int status, OAIPMH respBody) {
     } catch (Exception e) {
       logger.error("Exception marshalling XML", e);
     }
-    ctx.response().setStatusCode(status);
+
+    if (getErrorsProcessingConfigSetting().equals("200")) {
+      ctx.response()
+        .setStatusCode(200);
+    } else if (getErrorsProcessingConfigSetting().equals("500")) {
```

Comment:
yes, sure.

Reply:
On line 149 too please.

Reply:
The same code is in elseif and else. This can be shortened as
    if (getErrorsProcessingConfigSetting().equals("200")) {
      edgeResponse.setStatusCode(200);
    } else {
      edgeResponse.setStatusCode(httpStatusCode);
    }